module.json

```
@@ -32,37 +43,46 @@
 			"system": "pf2e",
 			"type": "Adventure",
 			"private": true,
-			"flags": {}
+			"ownership": {
+				"PLAYER": "LIMITED",
+				"ASSISTANT": "OWNER"
+			}
 		},
 		{
 			"label": "PF2e Magaambya Journals",
 			"name": "journals",
 			"path": "packs/journal.db",
 			"system": "pf2e",
 			"type": "JournalEntry",
-			"private": false,
-			"flags": {}
+			"ownership": {
+				"PLAYER": "LIMITED",
+				"ASSISTANT": "OWNER"
+			}
 		},
 		{
 			"label": "PF2e Magaambya Macros",
 			"name": "Macros",
 			"path": "packs/macros.db",
 			"system": "pf2e",
 			"type": "Macro",
-			"private": false,
-			"flags": {}
+			"ownership": {
+				"PLAYER": "LIMITED",
+				"ASSISTANT": "OWNER"
+			}
 		},
 		{
 			"label": "PF2e Magaambya Feats",
 			"name": "Feats",
 			"path": "packs/feats.db",
 			"system": "pf2e",
 			"type": "Item",
-			"private": false,
-			"flags": {}
+			"ownership": {
+				"PLAYER": "OBSERVER",
+				"ASSISTANT": "OWNER"
+			}
 		}
 	],
-	"url": "https://github.com/MrVauxs/pf2e-magaambya",
+	"url": "https://github.com/baronfel/pf2e-magaambya",

```

need to revert this, one sec.
```suggestion
	"url": "https://github.com/MrVauxs/pf2e-magaambya",

```